src/main/commandServer/__tests__/settingsValidator.spec.ts

```diff
@@ -151,17 +152,17 @@ describe(SettingsValidator, () => {
     });
 
     it('complains about mismatches between objects and scalars', () => {
-      let [needToUpdate, errors] = subject.validateSettings(cfg, { kubernetes: 5 });
+      let [needToUpdate, errors] = subject.validateSettings(cfg, { kubernetes: 5 as unknown as Record<string, number> });
```

Comment:
I'm telling TypeScript that `5` is really an object of some kind (which it obviously isn't), so that we can test the behaviour when we get invalid input.  (TypeScript is helpfully not letting us provide invalid input, of course.)

Reply:
Oh, the `as unknown as …` is to force TypeScript to forget what it knows about the thing on the left (because, obviously, manually casting `5` to `Record<string, number>` doesn't make any sense).